assets/styl/_settings/_color.styl

```diff
@@ -142,6 +143,7 @@ $color-flipboard ?= #f52828
 $color-github ?= #791e8f
 $color-googleplus ?= #d14836
 $color-hootsuite ?= #000000
+$color-instagram ?= #ff0040
```

Comment:
You forgot to add it in `$colors-socials` list here https://github.com/20minutes/colette/pull/324/files/d4fdcf3ac05ad1032c704859e634f663811454b8#diff-b245d9630074fe86b4c743811b49ebcdR157